platforms/ide/tooling-api/src/main/java/org/gradle/tooling/BuildController.java

```diff
@@ -293,7 +293,7 @@ <M, P> FetchModelResult<Model, M> fetch(
      * @since 9.3.0
      */
     @Incubating
-    <T extends Model, M, P> FetchModelResult<T, M> fetch(
+    <T extends Model, M, P> FetchModelResult<M> fetch(
```

Comment:
💅 Same here, I see that getModel and findModel has signature as:
```
<T, P> T findModel(Model target, Class<T> modelType, Class<P> parameterType, Action<? super P> parameterInitializer);
```
Can we make it the same here?